src/cart/crt_group.c

```diff
@@ -1849,6 +1849,35 @@ crt_group_config_path_set(const char *path)
 	return 0;
 }
 
+int
+crt_nr_secondary_remote_tags_set(int idx, int num_tags)
+{
+	struct crt_prov_gdata *prov_data;
+
+	D_DEBUG(DB_ALL, "secondary_idx=%d num_tags=%d\n", num_tags, idx);
```

Comment:
Looks like order on the parameters is reversed

Reply:
good catch, thanks